### PR TITLE
helix : add to auto updated github projects

### PR DESCRIPTION
**Pull request by MrAdityaAlok**
Suggested by @buttaface

### CODE REVIEW

File: scripts/updates/github-projects.txt [hunk 1]
```diff
@@ -737,3 +737,5 @@ zstd|facebook/zstd
 
 ## Returns 'null' in latest tag.
 #zziplib|gdraheim/zziplib
+
+helix/helix-editor/helix
```
Comment (buttaface) [inline scripts/updates/github-projects.txt:741]: Wrong format, first `/` should be a`|`.

Comment (buttaface) [reply scripts/updates/github-projects.txt:741]: Also, put it in alphabetical order in the file.

Comment (MrAdityaAlok) [reply scripts/updates/github-projects.txt:741]: > first / should be a |

Sorry! That was a typo 😄

File: scripts/updates/github-projects.txt [hunk 2]
```diff
@@ -181,6 +181,7 @@ gumbo-parser|google/gumbo-parser
 harfbuzz|harfbuzz/harfbuzz
 hashdeep|jessek/hashdeep
 helm|helm/helm
+helix|helix-editor/helix
```
Comment (buttaface) [inline scripts/updates/github-projects.txt:184]: Need to move it up one more, i before m.

Comment (MrAdityaAlok) [reply scripts/updates/github-projects.txt:183]: 😅 No word for such nonsense mistake!

**Pull request merged**